src/ur_ros_wrapper.cpp

```diff
@@ -344,7 +344,7 @@ class RosWrapper {
 
 		reorder_traj_joints(goal.trajectory);
 		
-		if (!start_positions_match(goal.trajectory, 0.01)) {
+		if (!start_positions_match(goal.trajectory, 0.04)) {
```

Comment:
Could you please change this to use a ros parameter with default `0.04` instead?
That way the user can adjust the limit in launch files if required.

For our arm, we had to increase this limit above `.08` and depending on the state of the arm other groups might need to increase it further.